spec/rspec/rails/matchers/action_cable/have_broadcasted_to_spec.rb

```diff
@@ -226,5 +226,34 @@ def broadcast(stream, msg)
         end
       end
     end
+
+    it "has an appropriate description when not qualified with a chain method" do
```

Comment:
Honestly I'd prefer leaving off the `when not ...` of all of these as the complexity would just grow over time.
```suggestion
    it "has an appropriate description" do
```